test/fuzz/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+options:
+  # make root approval non-recursive
+  no_parent_owners: true
+reviewers:
+  - sig-auth-api-reviewers
```

Comment:
oops, I had intended to suggest only sig-api-reviewers, but I see I put that in wrong in my suggestion in https://github.com/kubernetes/kubernetes/pull/124761#issuecomment-2189814578.  Would you be willing to correct?